app/addons/documents/docs-list/components.react.jsx

```diff
@@ -0,0 +1,119 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+define([
+  "api",
+  "react",
+  'addons/documents/docs-list/stores',
+  'addons/documents/docs-list/actions'
+  ], function (FauxtonAPI, React, Stores, Actions) {
+    var allDocsListStore = Stores.allDocsListStore;
+
+    var PerPageSelector = React.createClass({
+
+      perPageChange: function (e) {
+        var perPage = parseInt(e.target.value, 10);
+        this.props.perPageChange(perPage);
+      },
+
+      render: function () {
+        return (
+          <div id="per-page">
```

Comment:
if we want to build more small components in the future, we should try to avoid ids as selectors as the elements.

i think for this element it has no impact, just a thing i realized and wanted to mention